test/fuzz/fuzz_helpers.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
`#pragma once` is not portable -- the way to do this in halide is to surround with 

```
#ifndef HALIDE_FUZZ_HELPERS_H_
#define HALIDE_FUZZ_HELPERS_H_
...
#endif  // HALIDE_FUZZ_HELPERS_H_
```